core/sdk/src/main/java/com/mongodb/stitch/core/internal/net/StitchRequest.java

```diff
@@ -190,4 +191,40 @@ public StitchRequest build() {
           method, path, headers == null ? new HashMap<String, String>() : headers, body, startedAt);
     }
   }
+
+  protected Document toDocument() {
```

Comment:
[q] Why add this?

Reply:
For tests. It makes for nice assertEquals :)